core/build.gradle

```diff
@@ -94,6 +94,27 @@ android {
         }
     }
 
+    sourceSets {
+        prod {
+            def envMap = config.environments.find { it.key == "PROD" }
+            if (envMap.value.RES_DIR) {
```

Comment:
Maybe we can include a small comment in the README about using `RES_DIR` variable to include custom resources in the app?